gradle.properties

```diff
@@ -1,2 +1,2 @@
 group=com.marklogic
-version=1.0.0
+version=1.1.0
```

Comment:
I recommend changing this to 1.1-SNAPSHOT for now. 

Once you're ready to release 1.1, then add a separate commit that changes this to 1.1.0. Makes it really easy to look at commit history and see when the version changed. 

Also makes testing easier locally - i.e. you'd rather publish 1.1-SNAPSHOT locally than 1.1.0, as it's not clear that the latter is a non-released build.